lib/dom-actions.ts

```diff
@@ -140,6 +140,7 @@ export class DomActions implements DomActionsProvider {
             if (matches.length === 0) {
                 return [];
             }
+            // TODO: try all matches recursively?
```

Comment:
Was this comment intentional?

Reply:
yes, I'm not sure if this is worth doing, but I stumbled upon this implementation detail a few times, so decided to leave a comment